stable/hoard/README.md

```diff
@@ -0,0 +1,66 @@
+# Hoard
+
+[Hoard](https://github.com/monax/hoard) is a stateless, deterministically encrypted, content-addressed object store.
```

Comment:
Maybe add another sentence here about what hoard does.... In terms of utilizing various backends to store files that are encrypted or not. Also mentioning it's semantic encryption framework.